### DISCUSSION TITLE
Use client create method on table with Unsupported column type

### DISCUSSION THREAD

**Discussion by DrewVartanian** [prisma/prisma#7853] [Q&A] [answered]
I have a schema with a tsvector column, so I defined it as an Unsupported column type.

```prisma
model Things {
    id  String       @id @default(uuid()) @db.Uuid
    tsv Unsupported("tsvector")
}
```
Because of this column, I am unable to use the create method on the table's delegate.  `await prisma.things.create({data: thing })` creates the typescript error `Property 'create' does not exist on type 'ThingsDelegate<RejectOnNotFound | RejectPerOperation | undefined>'.`

I understand why if Prisma doesn't know how to handle the column, it would default to disabling the create method, but in my case, I have a Postgres function and trigger generating the data.  Is there any way to mark the column in the schema, so the client essentially ignores it, and allows create/upsert calls?  I only use the column in a raw query anyway, so I'm fine with losing it in any methods like find.  The `@ignore` doesn't seem to work with Unsupported, although I'm not sure if that would just affect the migration and not the client anyway.

Any solutions, insights, or workarounds are appreciated!

**Comment by ryands17** [comment, marked as answer]
@DrewVartanian 👋 

Unfortunately using raw queries for this model would be the only way to do this. Current we have a request to enhance this experience so you could add your use case here:
https://github.com/prisma/prisma/issues/5683

We also have a request for tsvector support so it would be great if you could add a 👍 to this so that we can look into the priority for this:
https://github.com/prisma/prisma/issues/5027

**Reply by DrewVartanian** [reply]
Thanks.  I gave each a 👍 , and I'll keep an eye on those.  It's too bad I can't just mark it so the client completely ignores the column in the meantime, like with the migration.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
